code/modules/gear_presets/survivors/solaris/crashlanding-offices_insert_bigred.dm

```diff
@@ -73,6 +75,23 @@
 	new_human.equip_to_slot_or_del(new /obj/item/stack/sheet/plasteel/med_small_stack(new_human), WEAR_IN_BACK)
 	new_human.equip_to_slot_or_del(new /obj/item/storage/belt/utility/full(new_human), WEAR_R_HAND)
 
+	..()
+// /obj/effect/landmark/survivor_spawner/bigred_crashed_pmc_leader
+
+/datum/equipment_preset/survivor/pmc/pmc_leader
```

Comment:
missing radio role title for Team Leader.

Add this under paygrades: role_comm_title = "TL"

Reply:
done, SL instead of TL for parity with ERT leads.